app/ai/voice/tts/gemini/service.py

```
@@ -0,0 +1,55 @@
+This is a thin override — once pipecat exposes ``CHUNK_SECONDS`` as a
+configurable parameter this module can be removed.
+"""
+
+from __future__ import annotations
+
+from pipecat.services.google.tts import GeminiTTSService
+
+__all__ = ["LowLatencyGeminiTTSService"]
+
+# Pipecat default is 0.5 s. We override to 0.1 s.
+# At 24 kHz × 2 bytes/sample, this is 4800 bytes per chunk (~100 ms of audio).
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)